pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1506,4 +1506,55 @@ self: super: {
   # through $PATH but they aren't in $PATH
   dhall-lsp-server = dontCheck super.dhall-lsp-server;
 
+  # hasn't bumped upper bounds
+  floskell = markUnbroken (dontCheck (doJailbreak super.floskell));
```

Comment:
markUnbroken is not optimal on the haskell-updates branch.

Better solution is to remove floskell from the broken list in configuration-hackage2nix.yaml.

For tests a quick comment on why it‘s unreasonable to fix in nixpkgs would be great (although not mandatory).

For jailbreaks an issue on upstream about bumping the bounds would be awesome.